jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/ImporterProcessor.java

```diff
@@ -72,15 +73,26 @@ public void processAndBackupFile(Path filePath, Path backupDir) {
       // ODE-559 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
       // ODE-559
+      boolean success = true;
       try (InputStream inputStream = new FileInputStream(filePath.toFile())) {
          BufferedInputStream bis = new BufferedInputStream(inputStream, odeProperties.getImportProcessorBufferSize());
          codecPublisher.publishFile(filePath, bis, fileType);
       } catch (Exception e) {
-         logger.error("Unable to open or process file: " + filePath, e);
+         success = false;
+         logger.error("Failed to open or process file: " + filePath, e);
```

Comment:
Since the same message is used in two places, it's better to define a String local variable. Use eclipse's `Refactor>Extract Local Variable` to do it for you.